docs/modules/ROOT/pages/release-notes.adoc

```diff
@@ -1,31 +1,35 @@
+* Add support for Topic CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/371[#371])
+* Add support for ReplicatedMap CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/394[#394])
+* Add support for CronHotBackup CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/362[#362])
+* Add support for restoring from HotBackup resource name (https://github.com/hazelcast/hazelcast-platform-operator/pull/401[#401])
+* Make Jet engine configurable from Hazelcast CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/457[#457])
+* Add Map Entry listener to get notified when a map entry is added, removed, updated, evicted, or expired (https://github.com/hazelcast/hazelcast-platform-operator/pull/408[#408])
+* Add support WAN Replication for multiple resources (https://github.com/hazelcast/hazelcast-platform-operator/pull/363[#363])
+* Add ClearWanQueue function into WanReplication finalizer executor (https://github.com/hazelcast/hazelcast-platform-operator/pull/403[#403])
+* Make LOGGING_LEVEL configurable for Hazelcast containers (https://github.com/hazelcast/hazelcast-platform-operator/pull/352[#352])
 
 == Changes
 
-* Enabled client-side code deployments (https://github.com/hazelcast/hazelcast-platform-operator/pull/340[#340])
-* Enabled JAR file uploads using ConfigMaps (https://github.com/hazelcast/hazelcast-platform-operator/pull/304[#304])
-* Enabled JAR file downloads into the Java classpath (PVC) (https://github.com/hazelcast/hazelcast-platform-operator/pull/244[#244])
-* Enabled a reference to the `Map` resource so that the resource is updated with configuration changes for WAN replication (https://github.com/hazelcast/hazelcast-platform-operator/pull/216[#216])
-* When a Hazelcast CR is marked for deletion, all dependent CRs are also deleted (https://github.com/hazelcast/hazelcast-platform-operator/pull/314[#314])
-* Added support for stopping hot backups gracefully (https://github.com/hazelcast/hazelcast-platform-operator/pull/318[#318])
-* Added configuration to skip checks of the Management Center lock file  (https://github.com/hazelcast/hazelcast-platform-operator/pull/333[#333])
-* Added Phone Home data collection as a separate process (https://github.com/hazelcast/hazelcast-platform-operator/pull/336[#336])
-* Added Phone Home data collection for Map CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/341[#341]), WAN replication (https://github.com/hazelcast/hazelcast-platform-operator/pull/342[#342]), user code deployment (https://github.com/hazelcast/hazelcast-platform-operator/pull/342[#342]), and backup and restore (https://github.com/hazelcast/hazelcast-platform-operator/pull/345[#345])
+* Remove the schedule from HotBackup API (https://github.com/hazelcast/hazelcast-platform-operator/pull/362[#362])
+* Add a default value to resource Kind in WanReplication spec (https://github.com/hazelcast/hazelcast-platform-operator/pull/374[#374])
```

Comment:
```suggestion
* Added a default value `kind` property value to the WanReplication CR specification (https://github.com/hazelcast/hazelcast-platform-operator/pull/374[#374])
```

Reply:
Probably this is better: 

Added a default `kind` property value to the WanReplication CR specification

Reply:
@angelasimms Is it ok to have the next: 
`Added a default value to the `kind` property in the `WanReplication` CR specification.`